src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt

```diff
@@ -1,17 +1,44 @@
 package org.vaccineimpact.orderlyweb.viewmodels
 
 import org.vaccineimpact.orderlyweb.ActionContext
+import org.vaccineimpact.orderlyweb.controllers.web.Serialise
 import org.vaccineimpact.orderlyweb.db.AppConfig
+import org.vaccineimpact.orderlyweb.db.Config
 import org.vaccineimpact.orderlyweb.models.Document
 
-data class DocumentsViewModel(val docs: List<Document>, val appViewModel: AppViewModel) : AppViewModel by appViewModel
+data class DocumentViewModel(val displayName: String,
+                             val path: String,
+                             val url: String,
```

Comment:
Previously we were building up the url in the template, so now generating it here as part of the serialised vm to be consumed by the vue component.